src/nvim/testdir/Makefile

```diff
@@ -160,7 +160,7 @@ nolog:
 # New style of tests uses Vim script with assert calls.  These are easier
 # to write and a lot easier to read and debug.
 # Limitation: Only works with the +eval feature.
-RUN_VIMTEST = $(TOOL) $(NVIM_PRG) -u unix.vim -U NONE -i viminfo --headless --noplugin
+RUN_VIMTEST = $(TOOL) $(NVIM_PRG) -u unix.vim
```

Comment:
removing `--headless` seems risky. That's for Nvim after ZyX improved the output.

Reply:
It is included via `$(NO_INITS)` now.

Reply:
where is `NO_INITS` defined?